src/ontology/cl-edit.owl

```diff
@@ -28067,9 +28069,13 @@ EquivalentClasses(obo:CL_2000031 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeV
 
 # Class: obo:CL_2000032 (peripheral nervous system neuron)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie"^^xsd:string) obo:IAO_0000115 obo:CL_2000032 "Any neuron that is part of a peripheral nervous system."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie"^^xsd:string) Annotation(oboInOwl:hasDbXref "ISBN:0721662544"^^xsd:string) obo:IAO_0000115 obo:CL_2000032 "Any neuron that is part of a peripheral nervous system."^^xsd:string)
```

Comment:
Is there a reason the wording "Any neuron..." is used instead of "A neuron..."?

Reply:
nope, really should be "a neuron" will make the change - didnt really look much into annotations when I was merging, good catch :)